src/proposals.cairo

```diff
@@ -204,9 +116,9 @@ mod Proposals {
         if index >= calldata_span.len() {
             return hashed_data;
         } else {
-            //let calldata_span: Span<(ContractAddress, u128)> = calldata.span();
             let (a, b) = *calldata_span.at(index);
-            let hashed_data = LegacyHash::hash(contract_address_to_felt252(a), b.into());
+            let b_felt: felt252 = b.into();
```

Comment:
why this change?

Reply:
Good catch. During the upgrade this wasn't compiling for some reason (unable to infer type annotations in parts of code not yet upgraded that also didn't compile) and this fixed it.

Done